addons/website/views/snippets/s_banner_connected.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_connected" name="Banner connected">
+    <section
+        class="s_banner_connected o_cc o_cc5 oe_img_bg pt128 pb136 parallax"
+        data-oe-shape-data="{'shape':'web_editor/Connections/13','colors':{'c5':'#FFFFFF'},'flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}"
```

Comment:
Here you added a `'showOnMobile':false` option while on line 17 you added a `o_shape_show_mobile` class. 

You will see, while being in the mobile preview of the editor, that the shape is visible on mobile while the mobile visibility option is set as "not visible" 

| Editor option | Mobile layout |
|--------|--------|
| ![Screenshot 2025-04-24 at 10 10 39](https://github.com/user-attachments/assets/2c4d0882-c7d1-4eb6-a500-b8bbe70d7b92) | ![Screenshot 2025-04-24 at 10 10 46](https://github.com/user-attachments/assets/a2a7ebaf-cc07-46e7-b052-881aeb7c930f) |